configure-bridgehead.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+useradd --home-dir /srv/docker/bridgehead -g docker -N -u 317 -s /sbin/nologin
```

Comment:
I would not set home-dir to /srv/docker/bridgehead. Use `-M` (don't create a home dir) option instead.
You would need to run `chown -R bridgehead:bridgehead /srv/docker/bridgehead` in the installation script after creating the user

Reply:
don't know if there is a better default for `-s /sbin/nologin`  to remove the option for login

Reply:
i wouldn't hardcode the userid to such a low number. Maybe we can find a better default

Reply:
How do you ensure, that this user is named "bridgehead"?